src/utils/curriculum/isVisibleUnit.ts

```diff
@@ -0,0 +1,24 @@
+import { YearSelection } from "@/components/CurriculumComponents/CurriculumVisualiser";
+import { CurriculumUnitsTabData } from "@/node-lib/curriculum-api-2023";
+
+export function isVisibleUnit(
```

Comment:
Not strictly a part of this work, just an FYI we also need some tests of this function. You can leave that for now though

Reply:
Added a placeholder test file for this and created a ticket in the backlog